File-Formats/PDF/Convert-HTML-To-PDF/azure.md

```diff
@@ -174,139 +162,134 @@ A complete work sample for converting an HTML to PDF in Azure App service on Lin
 
-Publish the docker image to Azure container instance.
+3. Publish the docker image to Azure container instance.
 ![Convert HTMLToPDF Azure Docker Step9](htmlconversion_images/AzureDocker5.png)
 
-It will push the docker image to the Azure container registry and deploy it to the Azure container instance.
+4. It will push the docker image to the Azure container registry and deploy it to the Azure container instance.
 ![Convert HTMLToPDF Azure Docker Step9](htmlconversion_images/AzureDocker6.png)
 
-After successful deployment, it will open the Azure website in the browser.
+5. After successful deployment, it will open the Azure website in the browser.
 ![Convert HTMLToPDF Azure Docker Step9](htmlconversion_images/AzureDocker6.png)
 
-Click Export to PDF option to convert the Google webpage to a PDF. You will get the PDF document as follows.
+6. Click Export to PDF option to convert the Google webpage to a PDF. You will get the PDF document as follows.
 ![Convert HTMLToPDF Azure Docker Output](htmlconversion_images/htmltopdfoutput.png) 
 
 A complete work sample can be downloaded from [BlinkLinuxDockerAzureSample.zip.](https://www.syncfusion.com/downloads/support/directtrac/general/ze/BlinkLinuxDockerAzureSample637072917)
 
 ## Azure App Function Linux
 
 ### Steps to convert HTML to PDF in the Azure Functions using the Blink rendering engine
-Create the Azure function project.
+1. Create the Azure function project.
 ![Convert HTMLToPDF Azure Functions Step1](htmlconversion_images/AzureFunctions1.png)
 
-Select the Azure Functions type and .NET Core version.
+2. Select the Azure Functions type and .NET Core version.
 ![Convert HTMLToPDF Azure Functions Step2](htmlconversion_images/AzureFunctions2.png)
 
-Install the [Syncfusion.HtmlToPdfConverter.Net.Linux](https://www.nuget.org/packages/Syncfusion.HtmlToPdfConverter.Net.Linux/) NuGet package as a reference to your .NET Core application [NuGet.org](https://www.nuget.org/)
+3. Install the [Syncfusion.HtmlToPdfConverter.Net.Linux](https://www.nuget.org/packages/Syncfusion.HtmlToPdfConverter.Net.Linux/) NuGet package as a reference to your .NET Core application [NuGet.org](https://www.nuget.org/)
 ![Convert HTMLToPDF Azure Functions Step3](htmlconversion_images/AzureFunctions3.png)
 
-Include the following namespaces in Function1.cs file.
+4. Include the following namespaces in Function1.cs file.
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-using Syncfusion.HtmlConverter;
-using Syncfusion.Pdf;
-using System.Runtime.InteropServices;
+   using Syncfusion.HtmlConverter;
+   using Syncfusion.Pdf;
+   using System.Runtime.InteropServices;
 
-{% endhighlight %}
+   {% endhighlight %}
 
-Add the following code snippet in the Function1 class to convert HTML to PDF using the Blink HTML converter in the Azure Functions Linux.
+5. Add the following code snippet in the Function1 class to convert HTML to PDF using the Blink HTML converter in the Azure Functions Linux.
```

Comment:
Modify - Add the following code snippet in the Function1 class to convert HTML to PDF using Blink rendering engine in the Azure Functions Linux.

Reply:
Modified